src/java.base/windows/native/libnio/ch/Net.c

```diff
@@ -786,3 +785,24 @@ Java_sun_nio_ch_Net_sendOOB(JNIEnv* env, jclass this, jobject fdo, jbyte b)
         return n;
     }
 }
+
+JNIEXPORT jboolean JNICALL
+Java_sun_nio_ch_Net_discardOOB(JNIEnv* env, jclass clazz, jobject fdo)
+{
```

Comment:
This seems surprisingly close to the logic of the drain method above (IOUtil.c). Would it make sense to factorize some of it?

Reply:
For now, this method sits sendOOB. There is some historical/legacy debt in this area in that some methods for network sockets have been added to IOUtil instead of Net. I do plan to re-visit this at some point but I'd prefer not do it in this PR because it requires touching all the Selector implementations.